README.md

```diff
@@ -52,7 +52,7 @@ Example RESTful interaction:
 | Receive test result data                                       |  GET ``results``                                          |
 | Receive test KPIs                                              |  GET ``kpi``                                              |
 | Receive test case name                                         |  GET ``name``                                             |
-
+| Receive forecast						 						 |  GET ``forecasts``					     				 |
 
```

Comment:
Additionally add the forecast directory and description to the Structure section of the README.md.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/commit/6fab1200752b6a0fb7a866a55cb20f5dfbf2e4e7.